site/src/components/card/Card.module.css

```diff
@@ -0,0 +1,60 @@
+.card {
+  --card-background: #10212b;
```

Comment:
If this color value doesn't exist in Odyssey's design tokens, we should maybe raise that with Shey. I'd be keen to understand if there is similar value from the design system we could use (or if perhaps the design system's palette needs to be extended to include this value).

In any case, it's fine for now :-)

Reply:
cc @shey-v this is in regards to the content blocks background colour